modules/server/src/topics/actionkeybindtopic.cpp

```diff
@@ -34,12 +34,56 @@ using nlohmann::json;
 
 namespace openspace {
 
-bool ShortcutTopic::isDone() const {
+bool ActionKeybindTopic::isDone() const {
     return true;
 }
 
-std::vector<nlohmann::json> ShortcutTopic::shortcutsJson() const {
-    std::vector<nlohmann::json> json;
+nlohmann::json jsonAction(const interaction::Action& action) {
+    // Convert to std as nlohmann doesn't understand glm
+    nlohmann::json colorJson;
+    if (action.color.has_value()) {
+        glm::ivec4 color = action.color.value();
+        // Convert to [0, 255] color range as that is the format the UI wants
+        color *= 255.0;
+        colorJson = { "color", { color.r, color.g, color.b, color.a } };
```

Comment:
But here, you should convert to ints before returning the value. '

However, My suggestion would be to maybe leave this conversion to 0-255 for the UI side, in case we want to use the serialization of the action colors for something else. Then, it would be better to not modify the data format for the color as part of the serialization.